index.js

```diff
@@ -251,6 +251,32 @@ const execa = (command, args, options) => {
 		return Promise.reject(error);
 	}
 
+	const originalKill = spawned.kill.bind(spawned);
+	spawned.kill = (signal, options) => {
```

Comment:
```suggestion
	spawned.kill = (signal, options = {}) => {
```

Then you can drop https://github.com/sindresorhus/execa/pull/208/files#diff-168726dbe96b3ce427e7fedce31bb0bcR263

Reply:
Fixed: a09b4a35609b4423ce14d5424539cd6e6c39f13f